reduce-todo/src/compo/Todo.js

```diff
@@ -0,0 +1,54 @@
+import React, {useState, useReducer} from "react";
+import TodoList from "./TodoList";
+import {reducer, initialState} from '../reducer/todoReducer'
+
+const Todo = () => {
+    const[newTodo, setNewTodo] = useState("Enter Todo!");
```

Comment:
Nice job handing state that's only in the component in the component. I agree with this decision.